tritium-lib/src/main/java/com/palantir/tritium/proxy/InvocationEventProxy.java

```diff
@@ -72,7 +72,7 @@ public String toString() {
     /** Optimized to avoid excessive stack frames for readable stack traces. */
     @Override
     @Nullable
-    public final Object invoke(Object proxy, Method method, @Nullable Object[] nullableArgs) throws Throwable {
+    public final Object invoke(Object proxy, Method method, Object @Nullable [] nullableArgs) throws Throwable {
```

Comment:
This is an example of what I described in my other comment as being confusing. And makes me wonder if places like https://github.com/palantir/tritium/blob/98661b54d2bdf0c8f2d51a25639f2c5bba61b47c/tritium-metrics/src/main/java/com/palantir/tritium/event/metrics/TaggedMetricsServiceInvocationEventHandler.java#L80-L80 are correct or not

Reply:
I think this should actually be:

```suggestion
    public final Object invoke(Object proxy, Method method, @Nullable Object @Nullable [] nullableArgs) throws Throwable {
```